runtimes/pythonrt/pythonrt.go

```diff
@@ -39,6 +39,19 @@ type pySvc struct {
 	comm      *Comm
 }
 
+var minPyVersion = Version{
```

Comment:
you already defined .python-version file. Could we ensure that this file is the single source of truth for min version?

Reply:
@tebeka unresolved back. let's talk?

Reply:
Removed `.python-version`